.rubocop.yml

```diff
@@ -70,6 +70,12 @@ Style/AccessorGrouping:
 Style/ArrayCoercion:
   Enabled: true
 
+Style/AsciiComments:
```

Comment:
what does this cop mean?

Reply:
@denys281  It offers to use ascii code instead of currency symbols. These symbols are used in config/initializers/money.rb